.github/CODEOWNERS

```diff
@@ -437,3 +437,5 @@ Makefile                                              @thiagokokada
 /modules/programs/ion.nix                             @jo1gi
 
 /modules/services/plex-mpv-shim.nix                   @starcraft66
+
+/modules/programs/tiny.nix                            @kmaasrud
```

Comment:
Could you move this line after `/modules/programs/timidity.nix @amesgen` to preserve the ordering as much as possible?

Reply:
Ofc! This should now be fixed 😄